.changeset/fresh-coins-shout.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Maybe we should have 1 changeset in this case, as all the changes here relate to enabling a specific feature.